lib/srfi/%3a176.mosh.sls

```diff
@@ -0,0 +1,11 @@
+;; this file is an alias-library.
+;;  alias of:
+;;   lib/srfi/%3a176/version-flag.sls
+(library (srfi :176)
+         (export
+             version-alist
+         )
+         (import
+             (srfi :176 version-flag)
```

Comment:
Should be `(srfi :176 version)`. I'll send a PR to add the `version` library name to [`srfi-data.scm`](https://github.com/scheme-requests-for-implementation/srfi-common/blob/master/admin/srfi-data.scm).

Reply:
Fixed. Thanks!